open_data_schema_map.pages.inc

```diff
@@ -419,17 +417,17 @@ function open_data_schema_map_add_type(&$values, $mapping) {
 }
 
 /**
- * Main endpoint callback
+ * Main endpoint callback.
  */
 function open_data_schema_map_endpoint($api) {
-  // Start by setting the content type header based on the API settings
+  // Start by setting the content type header based on the API settings.
   if ($output_format = open_data_schema_map_output_format_load($api->outputformat)) {
-    foreach($output_format['headers'] as $key => $value) {
+    foreach ($output_format['headers'] as $key => $value) {
       drupal_add_http_header($key, $value);
     }
   }
 
-  // Check if a file cache exists, and simply return this if it does
+  // Check if a file cache exists, and simply return this if it does.
   if (open_data_schema_map_file_cache_exits($api)) {
     $filename = open_data_schema_map_file_cache_name_helper($api);
     drupal_add_http_header('Expires', gmdate('D, d M Y H:i:s \G\M\T', time() + 3600));
```

Comment:
After displaying the cached endpoint I see the following errors on the next page request:

![image](https://user-images.githubusercontent.com/309671/30599809-8261a6aa-9d12-11e7-9f02-3dcbde577db6.png)

Reply:
Hmm.. I haven't been able to reproduce this yet :/